docs/src/content/docs/de/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,290 @@
+
+- `colors.accent` - wird für Links und die Hervorhebung des aktuellen Elements verwendet
+- `colors.gray` - wird für Hintergrundfarben und Rahmen verwendet
+- `fontFamily.sans` - wird für UI und Inhaltstext verwendet
+- `fontFamily.mono` - wird für Code-Beispiele verwendet
+
+```js {12,14,18,20}
+// tailwind.config.cjs
+const starlightPlugin = require('@astrojs/starlight-tailwind');
+const colors = require('tailwindcss/colors');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// Deine bevorzugte Akzentfarbe. Indigo entspricht am ehesten den Standardeinstellungen von Starlight.
+				accent: colors.indigo,
+				// Deine bevorzugte Grauskala. Zink kommt den Standardeinstellungen von Starlight am nächsten.
+				grau: colors.zinc,
+			},
+			fontFamily: {
+				// Deine bevorzugte Schriftart. Starlight verwendet standardmäßig eine Systemschriftart.
+				sans: ['"Atkinson Hyperlegible"'],
+				// Deine bevorzugte Code-Schriftart. Starlight verwendet standardmäßig die Systemschriftart Monospace.
+				mono: ['"IBM Plex Mono"'],
+			},
+		},
+	},
+	plugins: [starlightPlugin()],
+};
+```
+
+## Themes
+
+Das Farbtheme von Starlight kann gesteuert werden, indem die Standardeinstellungen überschrieben werden.
+Diese Variablen werden in der gesamten Benutzeroberfläche verwendet, wobei eine Reihe von Grautönen für Text- und Hintergrundfarben sowie eine Akzentfarbe für Links und zur Hervorhebung aktueller Elemente in der Navigation verwendet werden.
+
+### Farbdesign-Editor
+
+Verwende die Schieberegler unten, um die Akzent- und Graufarbpalette von Starlight zu ändern.
+Die dunklen und hellen Vorschaubereiche zeigen die resultierenden Farben, und die gesamte Seite wird ebenfalls aktualisiert, um deine Änderungen anzuzeigen.
+
+Wenn du mit deinen Änderungen zufrieden bist, kopiere den CSS- oder Tailwind-Code unten und verwende ihn in deinem Projekt.
+
+import ThemeDesigner from '../../../../components/theme-designer.astro';
+
+<ThemeDesigner
+	labels={{
+		presets: {
+			label: 'Voreinstellungen',
+			ocean: 'Ozean',
+			forest: 'Wald',
+			oxide: 'Oxid',
+			nebula: 'Nebel',
+			default: 'Standard',
+			random: 'Zufällig',
+		},
+		editor: {
+			accentColor: 'Akzent',
+			grayColor: 'Grau',
+			hue: 'Farbton',
+			chroma: 'Sättigung',
+			pickColor: 'Wähle Farbe',
+		},
+		preview: {
+			darkMode: 'Dark mode',
+			lightMode: 'Light mode',
+			bodyText:
+				'Der Fließtext wird in einem Grauton mit hohem Kontrast zum Hintergrund dargestellt.',
+			linkText: 'Links sind farbig.',
+			dimText:
+				'Einige Texte, wie z. B. das Inhaltsverzeichnis, haben einen geringeren Kontrast.',
+			inlineCode: 'Inline-Code hat einen eindeutigen Hintergrund.',
+		},
+	}}
+>
+	<Fragment slot="css-docs">
+		Füge das folgende CSS in deinem Projekt in einer [custom CSS
```

Comment:
`Custom `als Adjektiv wirkt hier, finde ich, etwas komisch. Wie fändest du diese Variante hier?
 
`Füge das folgende CSS in deinem Projekt in eine benutzerdefinierte CSS-Datei ein, um dieses Theme auf deiner Website anzuwenden.`

Und danke nochmal für diese grossartige Arbeit! Ich schau mir so bald es geht noch den Rest an.

Reply:
Nichts zu danken, ich bedanke mich für gründliche Untersuchung, damit die Dokumentation perfekt wird.

Die ganzen Kunjunktionsfehler kommen daher, dass der Deepl Übersetzer zuerst alles in die höfliche Form (Sie statt du) übersetzt hat und ich habe bereits erwartet, dass ich da einige Verben vergessen haben werde...

Deswegen nochmals danke für die ausführliche Korrektur, keep it going!